new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -372,6 +374,16 @@ public void testCreateMiiConfigMapDomain() {
         "patchDomainCustomResource(restartVersion)  failed ");
     assertTrue(rvPatched, "patchDomainCustomResource(restartVersion) failed");
 
+    ArrayList<String> pods = new ArrayList<>();
```

Comment:
get the pods list before patching domain

Reply:
moved it before patching

Reply:
fixed